txcache/testutils_test.go

```diff
@@ -36,6 +38,19 @@ type randomData struct {
 	itemSize    int
 }
 
+var defaultSelectionSessionMock = txcachemocks.SelectionSessionMock{
+	GetAccountStateCalled: func(address []byte) (state.UserAccountHandler, error) {
+		return &testscommonState.StateUserAccountHandlerStub{
+			GetBalanceCalled: func() *big.Int {
+				return big.NewInt(20)
```

Comment:
Not good for the long term (ultra-specific test object called "default", residing in generic test utilities).

However, since this mock is only used in one test, maybe define it there?

Reply:
Below, there's a function called `getTailItem`, it seems unused - should we drop it?